13-spatial-cv.Rmd

```diff
@@ -226,15 +229,18 @@ tm_shape(hs, bbox = bbx) +
 
 ```
 
-Here, when making predictions we neglect spatial autocorrelation since we assume that on average predictive accuracies of parametric models remain the same with or without spatial autocorrelation structures.
+Here, when making predictions we neglect spatial autocorrelation since we assume that on average the predictive accuracy remains the same with or without spatial autocorrelation structures.
+# Sp. autocor. structures can afaik only included during model fitting?
```

Comment:
universal kriging constructs a model and then estimates the spatial autocorrelation from the residuals. Finally, the interpolated residuals are added/subtracted from the predictions.